examples/Rasterizer/Rasterizer.cpp

```diff
@@ -2,34 +2,50 @@
 #include "pxEventLoop.h"
 #include "pxWindow.h"
 #include "pxKeycodes.h"
-#include "pxOffscreen.h"
-#include "pxTimer.h"
-#include "rtData.h"
+//#include "pxOffscreen.h"
+//#include "pxTimer.h"
+//#include "rtData.h"
 
-#include "xs_StringUtil.h"
+#define USE_PX_CANVAS
+//#define USE_XS_STRING
 
-#include "stdio.h"
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <fcntl.h>
+
+#ifdef USE_XS_STRING
 
 #ifndef PX_PLATFORM_WIN
-#include <unistd.h>
+  #include <unistd.h>
 #else
-#include <io.h>
-#define open _open
-#define close _close
-#define read _read
-#define fstat _fstat
-#define stat _stat
+  #include <io.h>
+  #define open _open
+  #define close _close
+  #define read _read
+  #define fstat _fstat
+  #define stat _stat
 #endif
 
-#define JOHNS
+  #include "stdio.h"
+  #include <sys/types.h>
+  #include <sys/stat.h>
+  #include <fcntl.h>
+
+  #include "xs_code/xs_StringUtil.h"
+#endif
 
-#ifdef JOHNS
-#include "pxCanvas.h"
+#ifdef USE_PX_CANVAS
```

Comment:
Code cleanup: Do we need to have the extra #if for USE_PX_CANVAS since it will always be defined?  Same question for other uses of #ifdef USE_PX_CANVAS.  If not then can you please remove the surrounding #ifdefs?